.github/workflows/tests.yml

```diff
@@ -12,29 +12,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ['2.6', '2.7', '3.0', '3.1', '3.2']
```

Comment:
let's keep all non EOL ruby and rails here since this is an open source project (ruby 3.0, 3.1, 3.2 and rails 6.1 and onward)